initial-configuration/install-dependencies-docker.sh

```diff
@@ -44,7 +43,28 @@ function set_docker_config_dir {
   echo 'alias picsure-db="docker exec -ti picsure-db bash -c '\''mysql -uroot -p\$MYSQL_ROOT_PASSWORD'\''"' >> "$rc_file"
 }
 
+function set_mysql_config_dir() {
```

Comment:
I wonder if this will cause confusion because the default value for the Jenkins global params is the root. I think both default values should be the same.